command/results_utils.go

```diff
@@ -91,7 +92,8 @@ func sendTestLogsAndResults(ctx context.Context, comm client.Communicator, logge
 	return nil
 }
 
-func sendTestResultsToCedar(ctx context.Context, t *task.Task, comm client.Communicator, results *task.LocalTestResults) error {
+// sendTestResultsToCedar sends the given test results to Cedar.
```

Comment:
I feel like the documentation here isn't necessary since the name is so clear and its an unexported function.

Reply:
Done.